tests/simple/testdata/proto2.textproto

```diff
@@ -1,8 +1,436 @@
+        { single_bool: true }
+      }
+    }
+  }
+
+  test {
+    name: "string"
+    container: "google.api.expr.test.v1.proto2"
+    expr: "TestAllTypes{single_string: 'foo'}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto2.TestAllTypes]
+        { single_string: 'foo' }
+      }
+    }
+  }
+
+  test {
+    name: "bytes"
+    container: "google.api.expr.test.v1.proto2"
+    expr: "TestAllTypes{single_bytes: b'\\377'}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto2.TestAllTypes]
+        { single_bytes: '\377' }
+      }
+    }
+  }
+}
+
+section {
+  name: "literal_wellknown"
+  description: "Literals with well-known fields set."
+
+  test {
+    name: "any"
+    container: "google.api.expr.test.v1.proto2"
+    expr: "TestAllTypes{single_any: TestAllTypes{single_int32: 1}}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto2.TestAllTypes] {
+        single_any {
+          [type.googleapis.com/google.api.expr.test.v1.proto2.TestAllTypes] {
+            single_int32: 1
+            }
+          }
+        }
+      }
+    }
+  }
+
+  test {
+    name: "duration"
+    container: "google.api.expr.test.v1.proto2"
+    expr: "TestAllTypes{single_duration: duration('123s')}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto2.TestAllTypes] {
+          single_duration {
+            seconds: 123
+          }
+        }
+      }
+    }
+  }
+
+  test {
+    name: "timestamp"
+    container: "google.api.expr.test.v1.proto2"
+    expr: "TestAllTypes{single_timestamp: timestamp('2009-02-13T23:31:30Z')}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto2.TestAllTypes] {
+          single_timestamp {
+            seconds: 1234567890
+          }
+        }
+      }
+    }
+  }
+
+  test {
+    name: "struct"
+    container: "google.api.expr.test.v1.proto2"
+    expr: "TestAllTypes{single_struct: {'one': 1, 'two': 2}}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto2.TestAllTypes] {
+          single_struct {
+            fields {
+              key: "one"
+              value { number_value: 1.0 }
+            }
+            fields {
+              key: "two"
+              value { number_value: 2.0 }
+            }
+          }
+        }
+      }
+    }
+  }
+
+  test {
+    name: "value"
+    container: "google.api.expr.test.v1.proto2"
+    expr: "TestAllTypes{single_value: 'foo'}"
+    value {
+      object_value {
+        [type.googleapis.com/google.api.expr.test.v1.proto2.TestAllTypes] {
+          single_value {
+            string_value: "foo"
+          }
+        }
+      }
+    }
+  }
+
+  test {
+    name: "int64_wrapper"
+    container: "google.api.expr.test.v1.proto2"
+    expr: "TestAllTypes{single_int64_wrapper: -321}"
```

Comment:
What happens on the wrappers if they're provided a `null` input? Is the result an empty `TestAllTypes`?

Reply:
Will test this in a future PR.